src/main/groovy/com/smartbear/swagger/Swagger2Importer.groovy

```diff
@@ -138,7 +138,7 @@ class Swagger2Importer implements SwaggerImporter {
 
         RestMethod method = resource.addNewMethod(opName)
         method.method = httpMethod
-        method.description = operation.summary
+        method.description = (operation.description ?: "").concat(System.getProperty("line.separator")).concat(operation.summary ?: "")
```

Comment:
could we add opposite processing when exporting? i.e. if there is a line-break in the description - put everything after the line-break in the summary?